sdks/java/extensions/avro/src/test/java/org/apache/beam/sdk/extensions/avro/schemas/utils/AvroUtilsTest.java

```diff
@@ -246,10 +247,17 @@ private static org.apache.avro.Schema getAvroSchema() {
             "bytes", org.apache.avro.Schema.create(Type.BYTES), "", (Object) null));
     fields.add(
         new org.apache.avro.Schema.Field(
-            "decimal",
-            LogicalTypes.decimal(Integer.MAX_VALUE)
-                .addToSchema(org.apache.avro.Schema.create(Type.BYTES)),
-            "",
+                "decimalBytes",
```

Comment:
I don't like here `LogicalTypes.decimal(Integer.MAX_VALUE)` replaced to decimals with prec&scale.  I think it's worth to keep existing test unless it's wrong completely.

Reply:
We should keep the original test case to make sure these changes don't break backwards compatibility